SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -25,7 +30,7 @@ final class SwedbankPaySDKViewModel: NSObject {
     private(set) var paymentOrder: SwedbankPaySDK.PaymentOrder?
     private(set) var consumerProfileRef: String?
         
-    var sessionManager: SessionManager = Alamofire.SessionManager(configuration: overrideUrlSessionConfigurationForTests ?? URLSessionConfiguration.default)
+    var sessionManager: Session = Alamofire.Session(configuration: overrideUrlSessionConfigurationForTests ?? URLSessionConfiguration.default)
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 142 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)